We are in git repository, but not in the timescaledb's repository.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -285,9 +285,7 @@
   message(FATAL_ERROR "Unable to find 'pg_config'")
 endif()
 
-find_package(Git)
-
-if(GIT_FOUND)
+if(FALSE)
   # We use "git describe" to generate the tag. It will find the latest tag and
   # also add some additional information if we are not on the tag.
   execute_process(
